src/containers/navigation/constants.tsx

```diff
@@ -37,17 +39,24 @@ export const EXT_MENU_OPTIONS = [
   },
 ];
 
-const CATEGORY_OPTIONS = [
-  { id: 'distribution_and_change', label: 'Distribution & Change', icon: STAR_SVG },
+const CATEGORY_OPTIONS: {
+  value: Category;
+  label: string;
+  icon: {
+    id: string;
+    viewBox?: string;
+  };
+}[] = [
+  { value: 'distribution_and_change', label: 'Distribution & Change', icon: STAR_SVG },
   {
-    id: 'restoration_and_conservation',
+    value: 'restoration_and_conservation',
     label: 'Restoration & Conservation',
     icon: RELOAD_SVG,
   },
-  { id: 'climate_and_policy', label: 'Climate & Policy', icon: SUN_SVG },
-  { id: 'ecosystem_services', label: 'Ecosystem Services', icon: CHART_SVG },
-  { id: 'contextual_layers', label: 'Contextual layers', icon: SUN_SVG },
-  { id: 'all_datasets', label: 'All datasets', icon: ALL_SVG },
+  { value: 'climate_and_policy', label: 'Climate & Policy', icon: SUN_SVG },
+  { value: 'ecosystem_services', label: 'Ecosystem Services', icon: CHART_SVG },
+  { value: 'contextual_layers', label: 'Contextual layers', icon: SUN_SVG },
+  { value: 'all_datasets', label: 'All datasets', icon: ALL_SVG },
```

Comment:
I think this was lost in the last rebase. "icon" field should be removed, and svgs removed from the platform

Reply:
label for "all_datasets" should be changed to "Custom"

so the file should be 

```
export const EXT_MENU_OPTIONS = [
  {
    id: 'about',
    label: 'About',
    href: 'https://www.mangrovealliance.org/about-us',
  },
  {
    id: 'mangroves',
    label: 'Mangroves',
    href: 'https://www.mangrovealliance.org/mangrove-forests',
  },
  {
    id: 'initiatives',
    label: 'Initiatives',
    href: 'https://www.mangrovealliance.org/initiatives',
  },
  {
    id: 'news',
    label: 'News',
    href: 'https://www.mangrovealliance.org/news',
  },
  {
    id: 'resources',
    label: 'Resources',
    href: 'https://www.mangrovealliance.org/tools-and-resources',
  },
  {
    id: 'contact',
    label: 'Contact',
    href: 'https://www.mangrovealliance.org/contact',
  },
];

const CATEGORY_OPTIONS = [
  { value: 'distribution_and_change', label: 'Distribution & Change' },
  {
    value: 'restoration_and_conservation',
    label: 'Restoration & Conservation',
  },
  { value: 'climate_and_policy', label: 'Climate & Policy' },
  { value: 'ecosystem_services', label: 'Ecosystem Services' },
  { value: 'contextual_layers', label: 'Contextual layers' },
  { value: 'all_datasets', label: 'Custom' },
];

export const STYLES = {
  'icon-wrapper': 'flex h-10.5 w-10.5 flex-col items-center justify-center text-brand-800',
};

export default CATEGORY_OPTIONS;
```